pkg/action/build_test.go

```diff
@@ -0,0 +1,122 @@
+package action
+
+import (
+	"github.com/helmwave/helmwave/pkg/helper"
+	"github.com/helmwave/helmwave/pkg/plan"
+	"github.com/helmwave/helmwave/pkg/repo"
+	"github.com/helmwave/helmwave/tests"
+	"github.com/urfave/cli/v2"
+	"os"
+	"testing"
```

Comment:
Maybe testify?

Reply:
testify is pretty. But I have thoughts  that is complicated/overhead for our goals